### DISCUSSION TITLE
Consideration of something like "institutional individual healthcare provider role"

### DISCUSSION THREAD

**Discussion by hoganwr** [ufbmi/OMRSE#258] [General]
Here, I am putting a discussion from monthly OMRSE meeting here on the GitHub repo. The discussion was about creating something like "institutional individual healthcare provider role" to focus our discussions on the roles people have within a healthcare organization. These roles are the output of deontic document acts (as defined by d-acts).

In researching this further, I found an existing class that seems to fit the bill, at least as a parent class for it:
current IRI: http://purl.obolibrary.org/obo/OMRSE_00000086
current label: human role within an organization
current definition: A human social role that is created by a deontic declaration performed by an organization.
current alternative label: human organizational role
current subClassOf: human social role

(note: current = 2024-12-26)

I propose creating a new class:
label: human role within a healthcare organization
definition: "A human role within an organization that is the bearer of a healthcare provider organization role"
subClassOf: human role within an organization

Note: there are other issues with the role hierarchy, which are logged as #99. My next move is to respond to the last comment there which is an analysis Clint made.

**Comment by dillerm** [comment]
Per my discussion with Bill today, I'm in favor of this class. If we don't hear any objections in the next week (by May 23), we will open an issue with a new term request.

**Comment by mbrochhausen** [comment]
I am in favor of this!Sent from my iPhoneOn May 16, 2025, at 9:59 AM, Matthew Diller ***@***.***> wrote:﻿
Per my discussion with Bill today, I'm in favor of this class. If we don't hear any objections in the next week (by May 23), we will open an issue with a new term request.
